src/network/networkd.h

```diff
@@ -145,6 +153,8 @@ struct Network {
         AddressFamilyBoolean ip_forward;
         bool ip_masquerade;
 
+        IPV6PrivacyExtensions ipv6_privacy_extensions;
```

Comment:
No, the "v" is lower case, "IP" is uppercase :)
https://en.wikipedia.org/wiki/IPv6